### ISSUE TITLE
create custom dataset

### ISSUE THREAD

**Issue by lana60-d** [davnords/LoMa#9]
Hello, thank you for the amazing work you've done.
I want to train or fine-tune a network using my own data, which consists only of paired images and the homography matrix between these two images.
Is something like this even possible?
Could you help me a bit?
In your opinion, would this work? I mainly want the network to be tuned with my data.

**Comment by davnords** [comment]
Hi @lana60-d. Thanks for your interest in our work.

Sorry about the slow reply.

I think this would work fine! You can then sample keypoints in both frames, transform them with the homography in loss.py, adapt mnn from depths to this context and then you should have GT assignments if I am not mistaken.

Is there anything I could assist you with? Happy to provide additional training code if needed. We also trained on flow datasets, not only depth ones.

/ David
